core/domain/exp_jobs_one_off.py

```diff
@@ -328,3 +328,25 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+class FallbackOneOffJob(jobs.BaseMapReduceJobManager):
+    """Job that outputs a list of fallbacks."""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_services.get_exploration_from_model(item)
+        for state_name, state in exploration.states.iteritems():
+            for fallback in state.interaction.fallbacks:
+                yield (state_name.encode('utf-8'),
```

Comment:
Regarding the test -- is it enough to create 2 explorations with two states containing 2 fallbacks with different num_submits and compare it with the output of this one-off job?